File: 준영/프로그래머스/LV2/JadenCase 문자열 만들기.js
function solution(s) {
  var answer = ''
  var idx = 0
  for (const i of s) {
    if (i === ' ') {
      answer += i
      idx = 0
      continue
    }

    if (idx === 0) {
      answer += i.toUpperCase()
      idx += 1
    } else {
      answer += i.toLowerCase()
    }
  }
  return answer
}

File: 준영/프로그래머스/LV2/JadenCase 문자열 만들기.py
def solution(s):
    answer = ''
    idx = 0
    for i in s:
        if i == ' ':
            answer += i
            idx = 0
            continue

        if idx == 0:
            answer += i.upper()
            idx += 1
        else:
            answer += i.lower()

    return answer

s = "3people unFollowed me"	
solution(s)
s = "aaaaa aaa "
solution(s)